plugins/node/opentelemetry-instrumentation-redis-4/README.md

```diff
@@ -17,8 +17,7 @@ npm install --save @opentelemetry/instrumentation-redis-4
 
 ### Supported Versions
 
-This package supports `redis@^4.0.0`
-For versions `redis@^2.6.0` and `redis@^3.0.0`, please use `@opentelemetry/instrumentation-redis`
+- [`redis`](https://www.npmjs.com/package/redis) versions `>=4.0.0`
```

Comment:
The instrumentation patches a different package then the user-facing one.

The patched packages are `@redis/client` and `@node-redis/client`, with version `^1.0.0` but the user-facing package has a different name and version `redis` with version 4.

I think otel users don't care about the internal packages and it's versioning, therefore I addressed `redis` in the README

Since we can't tell if a future major version of `redis` will continue using v1 of `@node-redis/client` or not, there is no way to tell the upper limit of the supported version for the 'redis' package itself